<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoJuego</class>
 <widget class="QMainWindow" name="NuevoJuego">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/sudoku1.png</normaloff>:/img/sudoku1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rBMedio">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Medio</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBDificil">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Difícil</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBFacil">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Fácil</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBJugar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/sudoku.png</normaloff>:/img/sudoku.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pBCancelar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/back.png</normaloff>:/img/back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBExperto">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Experto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{background-image:  url(:/img/TituloIco.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GrilledCheese BTN Toasted</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_2{font: 18pt &quot;GrilledCheese BTN Toasted&quot;;}
</string>
    </property>
    <property name="text">
     <string>Elija la dificultad:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbOpAlert">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GrilledCheese BTN Toasted</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Opciones de Alertas</string>
    </property>
    <widget class="QCheckBox" name="cbInvalidas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Alerta de Jugadas Invalidas</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbIncorrectas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Alerta de Jugadas Incorrectas</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbOpAyuda">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GrilledCheese BTN Toasted</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Ayuda</string>
    </property>
    <widget class="QCheckBox" name="cbhelp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Boton Ayuda</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
